### ISSUE TITLE
[SUGGESTIONS] [INTERFACE] Trades feed should display notifications about items being added at store for buying or selling

### ISSUE THREAD

**Issue by legionaw** [Eco-DiscordLink/EcoDiscordPlugin#22]
As the title says, the Trades feed should also notify about items being added to buying/selling lists at stores.

**Comment by MonzUn** [comment]
I'm not sure this can be done via a mod. I'll need to look into how those messages are fired.

**Comment by MonzUn** [comment]
Investigated this today. Implementing this requires creating a callback from the server. I'll make a pull request to Eco in the future to enable this feature.

**Comment by Oskar1504** [comment]
Wouldn't it be possible to store all tracked trades ( the store content) server side (bot) and run a cron job every 1h to check if something changed and send an update if so? 

Isn't the best way and an callback would be much easier and resource-saving but it could work, couldn't it.

**Comment by MonzUn** [comment]
Yeah, but that's a hacky solution that has quite a few cases where it won't work.
If you want that solution, you're free to fork and run that fork on your server.

**Comment by Oskar1504** [comment]
> Yeah, but that's a hacky solution that has quite a few cases where it won't work.
> If you want that solution, you're free to fork and run that fork on your server.

Hey thanks for answering. I read the source code from this mod and im not able to write an solution myself. I thought you'd using some different approach my idea wasn't that usable for the environment u build. Sry fpr that i commented to fast.
I read some other mod source code to forward the eco sever data to api endpoints. (Cost Calculator mod already created recipe, player.. endpoints).

Bit still thanks for the answer. Great mod u developed.

**Comment by MonzUn** [comment]
You don't have to apologize, but I also don't understand what you mean. It's not related to any endpoints at all.
